src/loading-bar.tsx

```diff
@@ -4,19 +4,29 @@ import {Portal} from './portal';
 import * as styles from './loading-bar.css';
 import {getPrefixedDataAttributes} from './utils/dom';
 import classnames from 'classnames';
+import {isRunningAcceptanceTest} from './utils/platform';
 
 import type {DataAttributes} from './utils/types';
 
 type Props = {visible: boolean; children?: void; dataAttributes?: DataAttributes};
 
 const LoadingBar: React.FC<Props> = ({visible, dataAttributes}) => {
     return (
-        <Portal className={classnames(styles.portal, {[styles.hidden]: !visible})}>
+        <Portal
```

Comment:
disable LoadingBar animations in acceptance tests

Reply:
hmmm... I think we already disabled all css animations with an injected css in acceptance tests

Reply:
Oh, I didn't know this. That's nice, I'll remove this logic then